commit 23834ede8ba891c56f609e204bd414b971cc6ca9 (refs/changes/95/2695/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2017-10-02T19:28:38+02:00 (2 years ago)
    
    TOOLS-1874 sdcadm should run with --abort-on-uncaught-exception
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/bin/sdcadm b/bin/sdcadm
index 2bf3566..f9d3c5e 100755
--- a/bin/sdcadm
+++ b/bin/sdcadm
@@ -6,13 +6,13 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 TOP=$(cd $(dirname $0)/../; pwd)
 if [[ -x "$TOP/build/node/bin/node" ]]; then
     # Dev layout.
-    $TOP/build/node/bin/node $TOP/lib/cli/index.js "$@"
+    $TOP/build/node/bin/node --abort_on_uncaught_exception $TOP/lib/cli/index.js "$@"
 else
-    /opt/smartdc/sdcadm/node/bin/node /opt/smartdc/sdcadm/lib/cli/index.js "$@"
+    /opt/smartdc/sdcadm/node/bin/node --abort_on_uncaught_exception /opt/smartdc/sdcadm/lib/cli/index.js "$@"
 fi
